packages/libs/error-stack/src/fmt.rs

```diff
@@ -99,7 +99,7 @@
 //!     .attach(Suggestion("try better next time!"))
 //!     .attach(Warning("unable to fetch resource"));
 //!
-//! # Report::set_color_mode(error_stack::fmt::ColorMode::Color);
+//! # Report::set_color_mode(error_stack::fmt::ColorMode::Emphasis);
```

Comment:
Can't we entirely remove this line? 🤔

Reply:
True, but this makes it easier to copy/paste the whole snippet around a couple of times w/o assuming any previous color mode set 😅